src/pages/documentation/migrate-to-android.md

```diff
@@ -16,18 +16,21 @@ At this time, the previous versions of the Android Mobile SDKs may be switched o
-| CampaignClassic | [![Maven Central](https://img.shields.io/maven-central/v/com.adobe.marketing.mobile/campaignclassic.svg?logo=android&logoColor=white&label=campaignclassic)](https://mvnrepository.com/artifact/com.adobe.marketing.mobile/campaignclassic)                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                  |
+| userprofile     | [![Maven Central](https://img.shields.io/maven-central/v/com.adobe.marketing.mobile/userprofile.svg?logo=android&logoColor=white&label=userprofile)](https://mvnrepository.com/artifact/com.adobe.marketing.mobile/userprofile)                                                                                                                                                                                                                                                                                                                                   |
+| assurance       | [![Maven Central](https://img.shields.io/maven-central/v/com.adobe.marketing.mobile/assurance.svg?logo=android&logoColor=white&label=assurance)](https://mvnrepository.com/artifact/com.adobe.marketing.mobile/assurance)    |
+| edge       | [![Maven Central](https://img.shields.io/maven-central/v/com.adobe.marketing.mobile/edge.svg?logo=android&logoColor=white&label=edge)](https://mvnrepository.com/artifact/com.adobe.marketing.mobile/edge) 
+| edgeidentity       | [![Maven Central](https://img.shields.io/maven-central/v/com.adobe.marketing.mobile/edgeidentity.svg?logo=android&logoColor=white&label=edgeidentity)](https://mvnrepository.com/artifact/com.adobe.marketing.mobile/edgeidentity) 
+| edgeconsent       | [![Maven Central](https://img.shields.io/maven-central/v/com.adobe.marketing.mobile/edgeconsent.svg?logo=android&logoColor=white&label=edgeconsent)](https://mvnrepository.com/artifact/com.adobe.marketing.mobile/edgeconsent) 
```

Comment:
can you also include the messaging and optimize to the list?

Reply:
Looks like we decided to keep all the SDK extensions. Following up on Emilia's comment here - please include messaging and optimize after Edge extensions but before the direct solution extensions.

Reply:
Updated.